Libraries/LibGfx/CompositingAndBlendingOperator.h

```diff
@@ -8,13 +8,14 @@
 
 namespace Gfx {
 
+// The values of this emun should match the constants in SVGFEBlendELement.idl
```

Comment:
IMO, if these need to match those exactly, they should all have an explicit value.

Reply:
Also typo: "emun"

Reply:
> IMO, if these need to match those exactly, they should all have an explicit value.

The issue is that only the values up to `CompositingAndBlendingOperator::Luminosity` correspond to IDL values in `SVGFEBlendElement` and setting explicit values for only some of the elements makes the linter complain.